DSpace/DSpace
# [FIXED] Deleting all the workflow groups too, when a collection is deleted.

## References
* This PR fixes issue #8063

## Description
No new functions are created in this PR.


## Instructions for Reviewers
### Steps to test
* Create a new collection.
* Edit the collection and create its workflow groups
* Delete the newly created collection.
* Check if the workflow groups are deleted as well.

List of changes in this PR:
* Code to delete all workflow groups has been added in `CollectionServiceImpl.java`.


## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_
